project_national/housing_characteristics/Insulation Unfinished Attic.tsv

```diff
@@ -1,451 +1,454 @@
+Multi-Family with 5+ Units	CR08	1970s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	1970s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	1970s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	1970s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR02	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR05	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR03	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR04	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR06	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR07	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR08	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	1980s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR02	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR05	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR03	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR04	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR06	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR07	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR08	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	1990s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR02	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR05	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR03	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR04	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR06	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR07	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR08	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	2000s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR02	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR05	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR03	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR04	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR06	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR07	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR08	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	1940s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR02	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR05	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR03	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR04	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR06	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR07	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR08	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR09	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR10	2010s	1	0	0	0	0	0	0	0
+Multi-Family with 5+ Units	CR11	2010s	1	0	0	0	0	0	0	0
```

Comment:
Still needs the decimal fix

Reply:
Fixed this one, committed and pushed. Were there others that showed up with the wrong decimals still?